indico/modules/events/surveys/templates/manage_survey.html

```diff
@@ -32,25 +32,66 @@
+    <div class="action-box">
+        <div class="section">
+            <div class="icon icon-info"></div>
+            <div class="text">
+                <div class="label">
+                    {% trans %}Submissions export{% endtrans %}
+                </div>
+                {% trans %}To export all the submissions, leave the checkboxes unchecked.{% endtrans %}
+            </div>
+        </div>
+    </div>
+
     <div class="i-box titled">
         <div class="i-box-header">
             <div class="i-box-title">
                 {% trans %}Submissions{% endtrans %}
             </div>
             <div class="i-box-buttons">
+                <button id="export-submissions" data-href="{{ url_for('.export_submissions', survey) }}"
+                        class="i-button" {{ 'disabled' if not survey.submissions }}>
+                    {% trans %}Export all submissions{% endtrans %}
+                </button>
                 <a href="#" class="i-button">
                     {% trans %}Show results{% endtrans %}
                 </a>
             </div>
         </div>
         <div class="i-box-content">
-            {% for submission in survey.submissions %}
-                <div>
-                    {{ submission }}
-                </div>
-            {% else %}
-                {% trans %}There are no submissions.{% endtrans %}
-            {% endfor %}
+            <ul class="group-list with-buttons no-content-before">
+                {%- for submission in survey.submissions %}
+                    <li class="submission-row">
+                        <strong>{{ _render_submission_row(submission) }}</strong>
+                        <span class="group right">
+                            <input class="inline-vcentered submission-ids" type="checkbox"
+                                   value="{{ submission.id }}">
+                            <a href="#" class="i-button icon-remove right js-delete-submission"
+                               data-href="{{ url_for('.delete_submission', submission) }}"
+                               data-method="DELETE"
+                               data-title="{% trans %}Remove submission?{% endtrans %}"
+                               data-confirm="{% trans %}Are you sure you want to remove this submission?{% endtrans %}">
+                                {% trans %}Delete{% endtrans %}
+                            </a>
+                        </span>
+                    </li>
+                {% endfor -%}
+                <li class="empty-message">{% trans %}There are no submissions.{% endtrans %}</li>
+            </ul>
         </div>
     </div>
+    <script>
+        setupSubmissionButtons();
+    </script>
 {% endblock %}
+
+{% macro _render_submission_row(submission) %}
+    <a href="{{ url_for('.display_submission', submission) }}">
+        {% if not submission.user -%}
+            {% trans %}Anonymous submission made on {% endtrans %} {{ submission.submitted_dt | format_datetime }}
```

Comment:
please don't use concatenation for this kind of thing - in another language you might need to put the date somewhere else.

    {% trans date=submission.submitted_dt | format_datetime %}Anonymous submission made on {{ date }}{% endtrans %}